types/material-ui/index.d.ts

```diff
@@ -1241,37 +1241,37 @@ declare namespace __MaterialUI {
         export interface IconMenuProps {
             // <Menu/> is the element that get the 'other' properties
             anchorOrigin?: propTypes.origin;
+            animated?: boolean;
             animation?: React.ComponentClass<Popover.PopoverAnimationProps>;
             className?: string;
-            iconButtonElement: React.ReactElement<IconButtonProps>;
+            clickCloseDelay?: number;
+            iconButtonElement: React.ReactElement<IconButton>;
             iconStyle?: React.CSSProperties;
             menuStyle?: React.CSSProperties;
-            onItemTouchTap?(e: React.SyntheticEvent<{}>, item: MenuItem): void;
+            onClick?(item: IconButton): void;
```

Comment:
The signature for `onClick` on the [docs](http://www.material-ui.com/#/components/icon-menu) is:  
`function(event: object) => void`  
*event*: Click event targeting the `IconButton` element.  

So I think this should be:  
`onClick?(e: React.SyntheticEvent<{}>): void;`

Reply:
@nbgraham You are right. I mistake to read the docs (the argument of `onClick` is `IconButton`).